src/bundle/Controller/LocationController.php

```diff
@@ -58,6 +58,13 @@ public function actionsAction(
                     return $this->resetLocation($content->id);
                 }
             }
+
+            if ($actionsForm->get('add')->isClicked()) {
```

Comment:
Would it be worth refactoring this and the deletion of location to a private method with a descriptive name to make it easier to read?